source/includes/fact-mongosh-shell-method-toc.rst

```diff
@@ -0,0 +1,18 @@
+The methods listed on this table of contents page are
+:mongosh:`mongosh </>` methods.  This is *not* the documention for
+``Node.js`` or other programming language specific driver methods.
+
+In most cases :mongosh:`mongosh </>` methods work the same way as the
+legacy :binary:`~bin.mongo` shell methods. However, some legacy methods
+are unavailable in ``mongosh``.
+
+For legacy :binary:`~bin.mongo` shell documention, refer to the
+documentation for the corresponding ``MongoDB`` release.
+
+   - :v4.4:`mongo shell v4.4 </mongo>`
```

Comment:
NB: This link is not yet live so it doesn't appear on the staging page. The link will be updated before the docs are released.